examples/RFToBModeGeneration.ipynb

```diff
@@ -0,0 +1,382 @@
+{
```

Comment:
<p>Do we still need to ignore warnings?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/ITKUltrasound/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>